detox/e2e/support/ui/screen.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {serverUrl} from '../test_config';
+
+// ****************************************************************
+// Select Server Screen
+// ****************************************************************
+
+/**
+ * fulfillSelectServerScreen enters server URL on Select Server screen and then tap connect button.
+ * @param {String} url - server URL
+ */
+export async function fulfillSelectServerScreen(url) {
+    await expect(element(by.id('select_server_screen'))).toBeVisible();
+
+    await element(by.id('server_url_input')).replaceText(url);
+
+    // # Tap anywhere to hide keyboard
+    await element(by.text('Enter Server URL')).tap();
+
+    await element(by.id('connect_button')).tap();
+}
+
+// ****************************************************************
+// Login Screen
+// ****************************************************************
+
+/**
+ * fulfillLoginScreen enters credential on Login screen and then tap "Sign in" button to log in.
+ * @param {Object} user - user to login with username and password
+ */
+export async function fulfillLoginScreen(user = {}) {
+    await expect(element(by.id('login_screen'))).toBeVisible();
+
+    await element(by.id('username_input')).replaceText(user.username);
+
+    // # Tap anywhere to hide keyboard
+    await element(by.text('Mattermost')).tap();
```

Comment:
only curious, do we have a reference somewhere what gestures are available? should we include them in the documentation even as a link?

Reply:
Yes, like swipe, pinch, scroll, etc - https://github.com/wix/Detox/blob/master/docs/APIRef.ActionsOnElement.md. Yes, definitely I'll add it to the docs.